source/Halibut.Tests/ServiceModel/PendingRequestQueueFixture.cs

```diff
@@ -236,6 +235,7 @@ public async Task QueueAndWait_AddingMultipleItemsToQueueConcurrently_AllRequest
         }
 
         [Test]
+        [NonParallelizable]
```

Comment:
I thought by default that no parallel execution takes place. So this might not be needed? Or is this more for telling future developers to not make it parallel?

Reply:
We have changed it to be parallel by default https://github.com/OctopusDeploy/Halibut/blob/main/source/Halibut.Tests/Properties/AssemblyInfo.cs